apps/emqx_retainer/etc/emqx_retainer.conf

```diff
@@ -58,9 +58,7 @@ retainer {
     batch_deliver_number = 0
 
     ## deliver limiter bucket
-    ##
-    ## Default: 0s
-    limiter_bucket_name = retainer
+    limiter.batch = retainer
```

Comment:
This seems a little hard to understand. Would this be better?
```
## The rate limiter name for retained messages delivery.
## In order to avoid delivering too many messages to the client at once,  which may cause the client
## to block or crash, or message dropped due to exceeding the size of the message queue. We need
## to specify a rate limiter for the retained messages delivery to the client.
##
## The names of the available rate limiters are taken from the existing rate limiters under `limiter.batch`.
## Default: retainer
batch_deliver_limiter = retainer
```